netmanager-app/core/apis/axiosConfig.ts

```
@@ -0,0 +1,42 @@
+  const axiosInstance = axios.create();
+
+  // Add a request interceptor
+  axiosInstance.interceptors.request.use(
+    (config) => {
+      if (isJWT) {
+        // Set the JWT authentication header
+        config.headers["Authorization"] = getJwtToken();
+        config.params = { ...config.params };
+      } else {
+        // Remove the JWT header and use a query parameter
+        delete config.headers["Authorization"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)